src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge())
+      .toBeInstanceOf(Array);
+  });
+
+  test('should return a numbers inside of array ', () => {
+    const result = getHumanAge(4, 4);
+
+    expect(typeof result[0]).toBe('number');
+    expect(typeof result[1]).toBe('number');
+  });
+
+  test('should have length 2 for input 2 args', () => {
+    expect(getHumanAge(1, 13))
+      .toHaveLength(2);
+  });
+
+  test('should return 14, 14 for 0, 0 inputs', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
```

Comment:
The test description states it should return `[14, 14]` for inputs `0, 0`, but the expected output is `[0, 0]`. Ensure the description matches the expected output.